views/apartments/edit.erb

```diff
@@ -0,0 +1,25 @@
+<form method="post" action="/apartments/<%=@apartment.id%>">
+  <input type="hidden" name="_method" value="put">
+
+  <label for="apartment[address]">Address</label>
```

Comment:
The `for` attribute should refer to the `id` attribute of an element, not the `name` attribute. This is because the `label` can only refer to one single `input` element, and there may be multiple `input` elements with the same name. For example, radio buttons:

```
<input type="radio" name="weekday" value="monday" />
<input type="radio" name="weekday" value="tuesday" />
...
```